libgnucash/engine/Account.h

```diff
@@ -1170,6 +1176,22 @@ gboolean xaccAccountGetPlaceholder (const Account *account);
  *  @param val The new state for the account's "placeholder" flag. */
 void xaccAccountSetPlaceholder (Account *account, gboolean val);
 
+/** Get the "opening-balance" flag for an account.  If this flag is set
+ *  then the account is used for opening balance transactions.
+ *
+ *  @param account The account whose flag should be retrieved.
+ *
+ *  @return The current state of the account's "opening-balance" flag. */
+gboolean xaccAccountGetOpeningBalance (const Account *account);
```

Comment:
Name these `xaccAccount[GS]etIsOpeningBalance`. `xaccAccount[GS]etOpeningBalance` suggests retrieving the balance itself.